telegram_bot.py

```diff
@@ -642,13 +643,13 @@ async def send_daily_async(nr_of_days:int):
                     day_profit = daily[item]['pnl']
                     pnl_sum += day_profit
                     previous_day_close_wallet_balance = wallet_balance - day_profit
-                    profit_pct = ((wallet_balance / previous_day_close_wallet_balance) - 1) * 100 \
+                    profit_pct = ((wallet_balance / previous_day_close_wallet_balance) - 1) / cross_wallet_pct * 100 \
```

Comment:
I think you should not divide by cross_wallet_pct here, but rather multiply. It's a value between 0 and 1, where 0.4 means that 40% of the balance is used.

Also, the way the bot works is that it simply multiplies the position['wallet_balance'] is multiplied by cross_wallet_pct. Wouldn't it be easier to multiply the wallet_balance by cross_wallet_pct?

Reply:
thank you for your feedback,

Division is the right operation for this case, because we are calculating the ratio (wallet_balance / previous_day_close_wallet_balance)  just before the operation.

On the other hand you have a point, it would be simpler to just multiply the wallet_balance by cross_wallet_pct, or maybe the right thing to do would be to introduce a new key to position dict named 'balance' and holding the value wallet_balance*cross_wallet_pct which would represent the effective balance the bot instance is working with?

That way we would use 'balance' to get accurate reports using commandes such as /daily /balance?

Reply:
seems to me neater to use wallet_balance * cross_wallet_pct instead of dividing by cross_wallet_pct after calculating the ratio

so something like
```
adjusted_wallet_balance = position['wallet_balance'] * float(self._bot.config['cross_wallet_pct'])
```